samples/multiplatform/src/wasmJsMain/resources/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Compose App</title>
+    <script type="application/javascript" src="skiko.js"></script>
+    <script type="application/javascript" src="composeApp.js"></script>
```

Comment:
Can we put these scripts before `</body>` tag? Loading scripts inside `head` tag is not recommended.